charmander/migrations/e107ab9eb193_initial_revision.py

```diff
@@ -0,0 +1,21 @@
+"""
```

Comment:
We use `alembic`. To be initialized properly, you need an initial revision which sets the `alembic` head to a specific version. The `cookiecutter` generates this revision here.